Source/WebCore/animation/WebAnimation.h

```diff
@@ -82,6 +82,9 @@ class WebAnimation : public RefCounted<WebAnimation>, public EventTarget, public
     virtual void setBindingsEffect(RefPtr<AnimationEffect>&&);
     AnimationEffect* effect() const { return m_effect.get(); }
     void setEffect(RefPtr<AnimationEffect>&&);
+
+    virtual AnimationTimeline* bindingsTimeline() const { return timeline(); }
+    virtual void setBindingsTimeline(RefPtr<AnimationTimeline>&&);
```

Comment:
I don't know how hot these are in benchmarks (presumably not that much, but please A/B test if there's a chance they are), but worth noting virtual functions are generally more costly

Reply:
This is the `Animation.timeline` property and to my knowledge it would not show in any of our benchmarks.